.github/workflows/regression-tests.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: logs
-          path: /tmp/failed/*
+          path: /tmp/*
```

Comment:
maybe limit to `/tmp/dragonfly.*`? there could be other, unrelated files under `/tmp`, no?

Reply:
so we will see, I am running regression tests to test that